nsls2api_lib.py

```diff
@@ -24,6 +24,35 @@ def get_all_cycles():
     return get_from_api(f"facility/nsls2/cycles")
 def get_proposals_from_cycle(cycle):
     return get_from_api(f"facility/nsls2/cycle/{cycle}/proposals")['proposals']
+
+# return true if any of check_instruments are in proposal_instruments
+def check_instruments_in_proposal(proposal_instruments, check_instruments):
+    for instrument in check_instruments:
+        if instrument.upper() in proposal_instruments:
+            return True
+    return False
+
+def get_proposals_for_cycle_instruments(cycle, instruments):
+    instruments_string = ""
+    for index, instrument in enumerate(instruments):
+        if index > 0:
+            instruments_string += "&"
+        instruments_string += f"beamline={instrument}"
```

Comment:
```suggestion
    instruments_string = "beamline=" + '&beamline='.join(instruments)
```
The above should do the same thing